test/gerkin/tests/features/data_read.feature

```diff
@@ -0,0 +1,15 @@
+Feature: API endpoint allowing users to search for one record in the Digital Registries database.
+  Request endpoint: POST /data/{registryname}/{versionnumber}/read
+
+  Background:
+    Given The user wants to search for one record in the Digital Registries database
+
+  Scenario: The user receives one searches record from the Digital Registries database
+    When The user triggers an action to search record in the database
+    And The request with a valid payload is sent
+    Then The user receives a searches record
+
+  Scenario: The user is not able to receive one searches record from the Digital Registries database
+    When The user triggers an action to search a record in the database
+    And The request with an invalid payload is sent
```

Comment:
It needs more information than just invalid payload. For example, the user will not receive any records even with a valid payload when the searched record doesn't exist in the database.